src/components/diagrams/networkAreaDiagram/network-area-diagram.js

```diff
@@ -271,6 +274,11 @@ const SizedNetworkAreaDiagram = (props) => {
                 studyUpdatedForce.eventData.headers['updateType'] === 'study'
             ) {
                 updateNad();
+            } else if (
+                studyUpdatedForce.eventData.headers['updateType'] ===
```

Comment:
This condition can be moved  in the first if condition with another `||`

Reply:
Done